Sources/WalletConnectRelay/RelayClient.swift

```diff
@@ -292,6 +317,8 @@ public final class RelayClient {
                     requestAcknowledgePublisherSubject.send(response.id)
                 } else if let subscriptionId = try? anyCodable.get(String.self) {
                     subscriptionResponsePublisherSubject.send((response.id, subscriptionId))
+                } else if let subscriptionIds = try? anyCodable.get([String].self) {
+                    batchSubscriptionResponsePublisherSubject.send((response.id, subscriptionIds))
```

Comment:
do we need to use different ack publishers? Can we use one?

Reply:
👍 we can